web-app/src/components/UserDetails/index.jsx

```diff
@@ -6,13 +6,14 @@ import { Card } from '../styled'
 
 export const UserDetails = (props) => {
 
+  const { user: { forename, surname, employeeRoleDescription } } = props;
   
   return (
     <Card>
         <strong>User Details</strong>
 
-        <div>Name: { props.user.surname + ', ' + props.user.forename }</div>
-        <div>Role: { props.user.employeeRoleDescription } </div>
+        <div><span>Name: { surname + ', ' + forename }</span></div>
```

Comment:
A little more ES6 syntax needed just `<div><span>Name: { `${surname}, ${forename}` }</span></div>`

You can use that for the others